lib/screen/clubs/clubs_screen.dart

```diff
@@ -29,41 +30,70 @@ class ClubsScreenState extends State<ClubsScreen> with BackNavigatorMixin, Error
+                return Stack(
+                  alignment: Alignment.center,
                   children: [
-                    ClubCard(
-                      onClick: () => viewModel.onRunningCardClicked(viewModel.runningClub),
-                      title: localization.clubSelectionRunningTitle,
-                      url: viewModel.runningPicture,
-                    ),
-                    const SizedBox(width: ThemeDimens.padding128),
-                    ClubCard(
-                      onClick: () => viewModel.onCyclingCardClicked(viewModel.cyclingClub),
-                      title: localization.clubSelectionCyclingTitle,
-                      url: viewModel.cyclingPicture,
+                    background,
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.center,
+                      children: [
+                        ClubCard(
+                          onClick: () => viewModel.onRunningCardClicked(viewModel.runningClub),
+                          title: localization.clubSelectionRunningTitle,
+                          url: viewModel.runningPicture,
+                        ),
+                        const SizedBox(width: ThemeDimens.padding128),
+                        ClubCard(
+                          onClick: () => viewModel.onCyclingCardClicked(viewModel.cyclingClub),
+                          title: localization.clubSelectionCyclingTitle,
+                          url: viewModel.cyclingPicture,
+                        ),
+                      ],
                     ),
                   ],
                 );
               } else {
-                return Column(
-                  mainAxisAlignment: MainAxisAlignment.center,
+                return Stack(
+                  alignment: Alignment.center,
                   children: [
-                    ClubCard(
-                      onClick: () => viewModel.onRunningCardClicked(viewModel.runningClub),
-                      width: 350,
-                      height: 350,
-                      title: localization.clubSelectionRunningTitle,
-                      url: viewModel.runningPicture,
+                    Image.asset(
+                      'assets/images/icappsGroup.jpg',
+                      fit: BoxFit.cover,
+                      width: double.infinity,
+                      height: double.infinity,
                     ),
-                    const SizedBox(height: ThemeDimens.padding56),
-                    ClubCard(
-                      onClick: () => viewModel.onCyclingCardClicked(viewModel.cyclingClub),
-                      width: 350,
-                      height: 350,
-                      title: localization.clubSelectionCyclingTitle,
-                      url: viewModel.cyclingPicture,
+                    const AnimatedGradiant(gradiantRatio: 0.2),
+                    Column(
+                      mainAxisAlignment: MainAxisAlignment.center,
```

Comment:
This is 90% the same widget as for a larger screen.. Maybe you should separate some parts and re-use them if possible.